cypress/e2e/list-management/related-links.feature

```diff
@@ -0,0 +1,53 @@
+Feature: List management change annual review date
+
+  Background:
+    Given I am logged in as a "Administrator"
+    And A "lawyers" list exists for Eurasia
+
+
+  Scenario: Links to related lists are shown
+    When a related lists exist
+    And I click the link "Lists"
```

Comment:
afaik logging in to list management the first page is the lists page. Is there any benefit to clicking on the Lists link?